{{cookiecutter.project_slug}}/.neuro/live.yml

```diff
@@ -0,0 +1,158 @@
+kind: live
+title: {{ cookiecutter.project_name }}
+id: PROJECT_SLUG_WITH_UNDERSCORES_PLACEHOLDER
+
+defaults:
+  preset: gpu-small-p
+  life_span: 1d
+  tags:
+    - kind:flow
+    - flow-id-with-dashes:{{ cookiecutter.project_slug }}
+
+volumes:
+  data:
+    remote: storage:$[[ flow.flow_id ]]/data
```

Comment:
`$[[ flow.flow_id ]]` is calculated to `live` string here and below.
Do you really need `flow_id`? Maybe `project_id` fits better?

Reply:
right! thanks

Reply:
actually, you can see in tests `test_live_yml.py` that `flow_id` is evaluated to `my_project`

Reply:
The test is not correct I guess.
Put `echo ${{flow.flow_id}}` for any job in live mode and execute it. You'll see `live` printout.